src/controllers/projectController.js

```diff
@@ -0,0 +1,151 @@
+const Document = require("../models/Document");
+
+const comparePages = require("../utils/comparePages");
+const saveFigmaDataAsNestedStructure = require("../utils/saveFigmaDataAsNestedStructure");
+
+exports.getAllVersions = async function (req, res) {
+  const { projectId } = req.params;
+  const accessToken = req.headers.authorization;
+
+  try {
+    const getVersions = await fetch(
+      `https://api.figma.com/v1/files/${projectId}/versions`,
+      {
+        method: "GET",
+        headers: {
+          "X-FIGMA-TOKEN": accessToken,
+        },
+      },
+    );
```

Comment:
```suggestion
const getAllVersions = async (req, res) => {
  const { projectId } = req.params;
  const accessToken = req.headers.authorization;

  try {
    const getVersions = await fetch(
      `https://api.figma.com/v1/files/${projectId}/versions`,
      {
        method: "GET",
        headers: {
          "X-FIGMA-TOKEN": accessToken,
        },
      },
    );
    
    
  
    ...
    
    
  module.exports = { getAllversions, getCommonPages };
```

- **`P2`**
1. 라우터 콜백함수는 arrow function으로 작성되어져 있는데 컨트롤러는 함수선언식으로 작성되어져 있어서
통일 해주면 좋을 것 같은데 혹시 어떻게 생각하시나요!

2. 현재는 함수를 선언할 때 prefix로 `exports.`을 붙혀주고 있는데 각각 함수선언과 `export`문을 분리해서 작성하면
조금 더 깔끔해 질 것 같습니다!
혹시 슬기님 생각은 어떠신가요?